apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -218,6 +221,10 @@ export class SendMessageEmail extends SendMessageBase {
       }
     }
 
+    if (overrides?.email?.replyTo) {
```

Comment:
overrides here is always empty because we do the `const overrides: Record<string, any> = command.overrides[integration?.providerId] || {};` command earlier.

Should it be :
```suggestion
    if (command.overrides?.email?.replyTo) {
```
?